prompt2model_demo.py

```diff
@@ -351,46 +353,113 @@ def main():
         evaluator_logger = get_formatted_logger("ModelEvaluator")
         evaluator_logger.setLevel(logging.INFO)
 
+        train_batch_size = None
+
         while True:
-            line = input("Enter the training batch size:")
+            line = input(
+                "Are you interested to train the model with automatic hyperparameter search? Type 'y' for Yes and 'n' for No. "  # noqa E501
+            )
             try:
-                train_batch_size = int(line)
-                assert 0 < train_batch_size
+                assert line in ["y", "n"]
                 break
             except Exception:
-                line_print("The training batch size must be greater than 0.")
+                line_print("The answer should be either y or n")
         time.sleep(1)
 
-        while True:
-            line = input("Enter the number of epochs to train for:")
-            try:
-                num_epochs = int(line)
-                break
-            except ValueError:
-                line_print("Invalid input. Please enter a number.")
-        time.sleep(1)
+        if line == "y":
+            line_print("Moving with training and hyperparameter selection.")
```

Comment:
Suggestion for cleaning up the language here:

```suggestion
            line_print("Starting training with hyperparameter selection.")
```

Reply:
@Anindyadeep could you accept my suggestion or revise this text otherwise? I think the current language sounds overly casual